todoapp/users/admin.py

```
@@ -1,3 +1,17 @@
 
-# Register your models here.
+from users import models as users_models
+
+
+class CustomUserAdmin(admin.ModelAdmin):
+    list_display = ("first_name", "last_name", "email", "date_joined")
+    search_fields = ("email", "first_name", "last_name")
+    list_filter = ("is_staff", "is_superuser", "date_joined")
+    ordering = ("-date_joined",)
+
+    def get_readonly_fields(self, request, obj = None):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_readonly_fields`

(ANN201)